wrench/src/main.rs

```diff
@@ -491,16 +498,44 @@ fn main() {
         let second_filename = subargs.value_of("second_filename").unwrap();
         perf::compare(first_filename, second_filename);
         return;
-    } else if let Some(subargs) = args.subcommand_matches("load") {
-        let path = PathBuf::from(subargs.value_of("path").unwrap());
-        let mut documents = wrench.api.load_capture(path);
+    } else {
+        panic!("Should never have gotten here! {:?}", args);
+    };
+
+    wrench.renderer.deinit();
+}
+
+fn render<'a>(
```

Comment:
nice moving logic out into standalone functions!
I wonder if we should make those to be methods of `Wrench`?

Reply:
I think that'd be a great idea. Perhaps a followup patch to keep the change as small as possible?